stack.yaml

```diff
@@ -1,13 +1,13 @@
-resolver: https://raw.githubusercontent.com/input-output-hk/cardano-prelude/c40175657bc8134e0561d9b6e6334728c19112e4/snapshot.yaml
+resolver: lts-14.22
 compiler: ghc-8.6.5
 
 packages:
   - cardano-config
   - cardano-node
 
 ghc-options:
-  cardano-config:   -Wall -Werror -fwarn-redundant-constraints
-  cardano-node:     -Wall -Werror -fwarn-redundant-constraints
```

Comment:
we explicitely wanted "-Werror"
why do you have to remove it?

Reply:
That snuck in during rebase, reverting..